source/develop/release-management/features/network/unrestricted-network-names.html.md

```diff
@@ -47,17 +47,19 @@ We will make no change in the Engine/Vdsm API. This makes backward compatibility
 
 A new field, "vdsm_name", is to be added to the network entity. When a new network is added, its existing "name" has to be provided by the human actor. The new "vdsm_name" is to be auto-generated by Engine according to a pattern
 
-      brXXXXXXXXXXXXX
+      onXXXXXXXXXXXXX
 
-where `XXXXXXXXXXXXX` are 13 hexadecimal characters extracted from the UUID of the network. "name" is to be editable by the end user, and exposed as the name of the network over REST (as it is today); "vdsm_name" would be immutable, and to be used only for Engine/Vdsm communication.
+where `XXXXXXXXXXXXX` are 13 hexadecimal characters extracted from the UUID of the network. "name" is to be editable by the end user, and exposed as the name of the network over REST (as it is today); "vdsm_name" would be immutable once the network is attached to a host and to be used only for Engine/Vdsm communication.
 
-### Upgrade
+In a scenario where "name" doesn't exceed IFNAMSIZ, in order to ease debugging and backwards compatibility, "vdsm_name" will match "name". In order to avoid any lingering states that
+may be resulted in network name changes, we'll continue disallowing changing network names of networks that are host attached.
 
-On upgrade, the "name" of existing networks would be copied into their "vdsm_name". REST-based scripts and GUI users would not see a difference. However, they could now edit "name" with no Linux-based limitations.
+As aforementioned, Engine/Vdsm communication will now be made via "vdsm_name", while all intra Engine communication will remain via "name". The discrepancy will be filled by using a "name" and
```

Comment:
Intra engine communication is usually id based, I think the access by `name` is used only in one place, so I would expect that communication inside engine is not relevant here.